cdk/lib/mobile-save-for-later.ts

```diff
@@ -144,8 +147,5 @@ export class MobileSaveForLater extends GuStack {
         },
       ],
     });
-
-    // TODO:
-    // Decide whether to port across or leave in CFN: Dynamo Table & Dynamo Throttle CloudWatch Alarms
```

Comment:
Minor: it might be worth creating a card to migrate the Dynamo table into TypeScript now, as this is the only thing left in the YAML file? I don't think it's especially high priority though.

Reply:
👍 good idea